cloudwash/providers/aws.py

```diff
@@ -11,27 +11,44 @@
 def cleanup(**kwargs):
     is_dry_run = kwargs.get("dry_run", False)
     regions = settings.aws.auth.regions
-    if "all" in regions:
-        with compute_client("aws", aws_region="us-west-2") as client:
-            regions = client.list_regions()
-    for region in regions:
-        for items in data:
-            dry_data[items]['delete'] = []
-        with compute_client("aws", aws_region=region) as aws_client:
-            awscleanup = AWSCleanup(client=aws_client)
-            # Actual Cleaning and dry execution
-            logger.info(f"\nResources from the region: {region}")
-            if kwargs["vms"] or kwargs["_all"]:
-                awscleanup.vms.cleanup()
-            if kwargs["nics"] or kwargs["_all"]:
-                awscleanup.nics.cleanup()
-            if kwargs["discs"] or kwargs["_all"]:
-                awscleanup.discs.cleanup()
-            if kwargs["pips"] or kwargs["_all"]:
-                awscleanup.pips.cleanup()
-            if kwargs["images"] or kwargs["_all"]:
-                awscleanup.images.cleanup()
-            if kwargs["stacks"] or kwargs["_all"]:
-                awscleanup.stacks.cleanup()
-            if is_dry_run:
-                echo_dry(dry_data)
+    if kwargs["ocps"]:
+        aws_client_region = settings.aws.criteria.ocps.ocp_client_region
+        with compute_client("aws", aws_region=aws_client_region) as aws_ocp_client:
+            if "all" in regions:
+                regions = aws_ocp_client.list_regions()
+            awscleanup = AWSCleanup(client=aws_ocp_client)
+            for region in regions:
+                aws_ocp_client.cleaning_region = region
+                # Emptying the dry data for previous region everytime
+                for items in data:
+                    dry_data[items]['delete'] = []
+                logger.info(f"\nResources from the region: {region}")
+                awscleanup.ocps.cleanup()
+                if is_dry_run:
+                    echo_dry(dry_data)
+    else:
```

Comment:
Might not want to use `else` since the cleanup can be set to execute OCPs and other resources as well (or use `kwargs["_all"]`)

Reply:
Do you see a scenario where OCPs and other resources in the list would be called at the same time for cleanup. ?

If yes, we should rather say `ocp-clusters` to be more specific, so it would look like a resource type within AWS provider.

I thought it is good to keep OCP cluster cleanup separate from other resources cleanup and cluster cleanup will anyways cleanup what it suppose to.

Thoughts ?